src/application/wizard/SetupPage.tsx

```diff
@@ -116,7 +132,7 @@ export class SetupPage extends Component<Props, State> {
               Next we are going to send a test notification. Make sure you {''}
               <Text
                 component={TextVariants.a}
-                href="https://aerogear.org/docs/unifiedpush/aerogear-push-android/guides/#_handling_notification"
+                href={getLink('build-and-deploy-android')}
```

Comment:
`build-and-deploy-android` seems to be used for all variants (android, ios, etc) should it be renamed?